src/_data/services.yml

```diff
@@ -0,0 +1,51 @@
+industry: Solutions Boutique
+intro: |
+  We’ve been around the block before and we’ve sat on the other side of the table.  We won’t waste your time or money with countless meetings and processes.  In fact, if it’s not work on a deliverable we *don’t charge*.   That means all of our work is peer reviewed by another team member *at no charge to our clients*.  Why?  Because we believe in delivering the right solution, the first time
+
+area-served:
+  type: Text # valid: Text, or Place (e.g. State, or City)
+  caption: Worldwide
+catalog:
+  - name: Product Design # link caption
+    slug: digital-product-design # link url
+    title: Digital Product Design # page title
+    description: Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum
```

Comment:
Do we need to fill in a `description` for all of these in the file?

Reply:
You can leave them blank, but since these descriptions pertain to SEO, I would have to advise against it. 

As a side note, this catalog is pretty much pulled directly from other agency sites. We should start completely fresh with our own catalog.